src/subtasks/configure.js

```diff
@@ -30,8 +33,10 @@ module.exports = (logger, cwd, config) => {
     yaml: ['.travis.yml', '.circleci/config.yml'],
     seq: ['.gitignore', '.npmignore']
   };
-  const filteredTasks = Object.keys(tasks)
-    .reduce((p, c) => Object.assign(p, { [c]: tasks[c].filter(f => toSkip.indexOf(f) === -1) }), {});
+  const filteredTasks = Object.keys(tasks).reduce(
+    (p, c) => Object.assign(p, { [c]: tasks[c].filter(f => toSkip.indexOf(f) === -1) }),
+    {}
+  );
```

Comment:
What's the logic behind the refactoring here? Because it has an anonymous function inside the `reduce`? I feel like some readability has been lost with this change

Reply:
All prettier. Can't influence that